railties/lib/rails/generators/rails/app/templates/Gemfile

```diff
@@ -36,12 +36,21 @@ group :development, :test do
 end
 
 group :development do
+<%- if options.api? -%>
+  # Access an IRB console on exception pages
+  <%- if options.dev? || options.edge? -%>
+  # gem 'web-console', github: 'rails/web-console'
```

Comment:
Why we are even adding web-console commented? If the app is API only we will not care about web-console